helper/helper_test.go

```diff
@@ -24,6 +174,50 @@ var _ = Describe("helper tests", func() {
 		Options:        make(map[string]string),
 	}
 
+	BeforeEach(func() {
+		saveBackupAgent = *backupAgent
```

Comment:
Why not just:
```
		*backupAgent = <default value>
		*compressionLevel = <default value>
		*compressionType = <default value>
		*content = <default value>
		*dataFile = <default value>
		*oidFile = <default value>
		*onErrorContinue = <default value>
		*pipeFile = <default value>
		*pluginConfigFile = <default value>
		*printVersion = <default value>
		*restoreAgent = <default value>
		*tocFile = <default value>
		*isFiltered = <default value>
		*copyQueue = <default value>
		*singleDataFile = <default value>
		*isResizeRestore = <default value>
		*origSize = <default value>
		*destSize = <default value>
		*verbosity = <default value>
```
in BeforeEach() and remove AfterEach()?
Thus we can reduce code size and make the code more straightforward for understanding.

Reply:
Global values initialized in InitializeGlobals() as a command line arguments. In this approach I am trying to save that values and have them in any test. If particular test need specific hardcoded value, it will override it, but global one will be restored in AfterEach().
It adds some flexibility, as we can potentially override defaults from command line.

Reply:
Hm, I'm not sure wether we can easily control the values from the command line within our unit test infrastructure. Can you provide an example of scenario to demonstrate the use case you are worried about?

Reply:
I don't have such a scenario either, I just want not to loose this possibility, by the cost of isolated few lines of code. Current infrastructure can be changed at any moment, we will be ready for it.

Reply:
I think it is excessive. Please involve 3rd opinion.

Reply:
@RekGRpth - what do you think?

Reply:
I think we shouldn't worry about global variables. These are [unit tests that are run separately on GitHub](https://github.com/arenadata/gpbackup/actions/runs/10775138265/job/29878872662?pr=102#step:7:54), especially since no one bothered with the existing "Check subset flag" tests.

Reply:
Okay